tests/Unit/CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ target_link_libraries(
   Options
   SystemUtilities
   Utilities
+  util
```

Comment:
I can't find this target anywhere. What is it?

Reply:
I put it in because when I first tried this on Caltech HPC the linker failed because `libutil` was needed for something in Pypp, I think.
Now it no longer fails without this, likely because other linker problems were fixed and this links in via other dependencies -- I'll remove this and fix the `Python_EXECUTABLE` thing in a follow-up.